README.md

```diff
@@ -49,18 +49,22 @@ Hello welcome to my **[Portfolio]**, here you'll find details about me and some
 
 ### Key Features <a name="key-features"></a>
 
-- Toolbar section
-- Portfolio section
-- About Me section
-- Contact Me section
+- Home Page
+- About page
+<!-- - Contact Me section -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Video Demo <a name="video"></a>
+
+- A short video walk-through of the [website](https://loom.com/share/b04d73d12cff43fd982fce9faba18320)
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="https://jeddaa.github.io/microverse-website-one/"></a>
+## 🚀 Live Demo <a name="https://jeddaa.github.io/microverse-project1/"></a>
+
+<!-- If you prefer to see the finished version of the project visit the [live site](https://jeddaa.github.io/microverse-website-one/) -->
 
-If you prefer to see the finished version of the project visit the [live site](https://jeddaa.github.io/microverse-website-one/)
+The finished project will be deployed very soon.
```

Comment:
- [ ] Great job on creating the README file for your project! :+1: However, it's important to ensure that you deploy your project and add the link to the live version in your README file, even if you haven't finished your project yet. This is important because it will allow others to easily access and view your project, as well as provide you with a way to test your project on a live server. Additionally, it will be automatically updated with every change that you push to your GitHub repository. Therefore, it's important to deploy your project and add the link to the live version in your README file to provide easy access to your project.